CMakeLists.txt

```diff
@@ -863,3 +864,19 @@ IF ("${VERSION_PRERELEASE}" STREQUAL "-DEV" AND EXISTS "${CMAKE_SOURCE_DIR}/.git
     ENDIF (LIBUV_FOUND)
 ENDIF ()
 
+
+OPTION(WITH_H2OLOG "use h2olog(1)" OFF)
```

Comment:
Rather than defaulting to "OFF", I think it might be better if we set this to on whenever the prerequisites are met. Like how we handle WITH_MRUBY (and WITH_MRUBY_DEFAULT).

Reply:
Done.  `-DWITH_H2OLOG=ON` will be set if dependencies are satisfied, so `label=dtrace` build in CI makes `h2olog`.